packages/frontend/src/utils/getSearch.ts

```diff
@@ -0,0 +1,13 @@
+import { IUser } from 'src/types';
+
+export const getSearch = (users: IUser[], value?: string) => {
```

Comment:
I would change it to request to Firestore instead of making filtering on frontend side, because we make app with the idea of scalability in mind since UVA team said they want to add more countries in the future, so there'll be more people. And while searching in frontend works faster it may start causing problems where there'll be 100+ people.

I've found some example of how such search can be done here:

https://stackoverflow.com/questions/46568142/google-firestore-query-on-substring-of-a-property-value-text-search
```javascript
collectionRef
    .where('name', '>=', value)
    .where('name', '<=', value + '\uf8ff')
```

And also some sample from chatGPT, not sure it really works, but can be used as some example:
```javascript
import { collection, query, where, getDocs, orderBy, startAt, endAt } from 'firebase/firestore';

// Предполагается, что db - это ссылка на ваш Firestore экземпляр
const db = firebase.firestore();

const getUsersByName = async (name) => {
    // Коллекция, в которой хранятся пользователи
    const usersRef = collection(db, 'users');
    
    // Создаем запрос для поиска пользователей, имя которых начинается с введенных символов
    // Для этого используем orderBy и startAt для указания начального значения
    // и endAt для указания конечного значения, добавляя '\uf8ff' для включения всех символов после введенных
    const queryStartsWith = query(usersRef, orderBy('name'), startAt(name), endAt(name + '\uf8ff'));
    
    // Выполняем запрос
    const querySnapshot = await getDocs(queryStartsWith);
    
    // Массив для хранения результатов
    const usersList = querySnapshot.map((doc) => doc.data());
    
    return usersList;
};
```